mmhuman3d/core/visualization/visualize_smpl.py

```diff
@@ -247,32 +247,30 @@ def _prepare_mesh(poses, betas, transl, verts, start, end, body_model):
                 raise KeyError(
                     f'{str(poses.keys())}, Please make sure that your '
                     f'input dict has all of {", ".join(body_pose_keys)}')
-            num_frame = poses['body_pose'].shape[0]
+            num_frames = poses['body_pose'].shape[0]
             _, num_person, _ = poses['body_pose'].view(
-                num_frame, -1, NUM_BODY_JOINTS * 3).shape
+                num_frames, -1, NUM_BODY_JOINTS * 3).shape
 
             full_pose = body_model.dict2tensor(poses)
-            start = (min(start, num_frame - 1) + num_frame) % num_frame
-            end = (min(end, num_frame - 1) + num_frame) % num_frame
-            full_pose = full_pose[start:end + 1]
+            start = (min(start, num_frames - 1) + num_frames) % num_frames
+            full_pose = full_pose[start:end]
 
         elif isinstance(poses, torch.Tensor):
             if poses.shape[-1] != NUM_DIM:
                 raise ValueError(
                     f'Please make sure your poses is {NUM_DIM} dims in'
                     f'the last axis. Your input shape: {poses.shape}')
             poses = poses.view(poses.shape[0], -1, (NUM_JOINTS + 1) * 3)
-            num_frame, num_person, _ = poses.shape
-            start = (min(start, num_frame - 1) + num_frame) % num_frame
-            end = (min(end, num_frame - 1) + num_frame) % num_frame
-            full_pose = poses[start:end + 1]
+            num_frames, num_person, _ = poses.shape
+            start = (min(start, num_frames - 1) + num_frames) % num_frames
```

Comment:
Similar as above.

Reply:
this is still useful since 'start_frame' and `end_frame` must be None.